### ISSUE TITLE
mypy CI job is broken

### ISSUE THREAD

**Issue by pmeier** [pytorch/vision#7522]
For example: https://github.com/pytorch/vision/actions/runs/4719517927/jobs/8370432599?pr=5009#step:10:824

```python
# roll.py
import torch

torch.empty(3).roll(1)
```

```
❯ python -c "import torch; print(torch.__version__)"
2.1.0.dev20230407+cpu
❯ mypy roll.py
Success: no issues found in 1 source file
```

```
❯ python -c "import torch; print(torch.__version__)"
2.1.0.dev20230417+cpu
❯ mypy roll.py
roll.py:3: error: Argument 1 to "roll" of "_TensorBase" has incompatible type "int"; expected "Sequence[Union[int, SymInt]]"  [arg-type]
Found 1 error in 1 file (checked 1 source file)
```

Probable culprit: pytorch/pytorch#99114

cc @seemethere

**Comment by pmeier** [comment]
Maybe fixed by pytorch/pytorch#99921. Let's see if that makes it into today's nightly or we have to wait until tomorrow.

**Comment by pmeier** [comment]
Fixed now.
